ethash/src/progpow.rs

```diff
@@ -14,6 +14,22 @@
 // You should have received a copy of the GNU General Public License
 // along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 
+//! ProgPoW (Progressive Proof-of-Work) is the Ethereum network's proposed new Application-Specific Integrated
```

Comment:
```suggestion
//! ProgPoW (Programmatic Proof-of-Work) is the Ethereum network's proposed new Application-Specific Integrated
```

https://github.com/ifdefelse/ProgPOW